internal/store/endpoint_test.go

```diff
@@ -93,8 +168,11 @@ func TestEndpointStore(t *testing.T) {
 		},
 	}
 	for i, c := range cases {
-		c.Func = generator.ComposeMetricGenFuncs(endpointMetricFamilies)
-		c.Headers = generator.ExtractMetricFamilyHeaders(endpointMetricFamilies)
+		allowLabels := []string{
```

Comment:
Diff is strange here, so best expand, essentially this adds a Labels test, with labels we allow for the kube_endpoint_labels metric.